Java/corenative/src/main/java/com/breadwallet/corenative/crypto/BRCryptoTransferState.java

```diff
@@ -30,21 +30,25 @@
             public long transactionIndex;
             public long timestamp;
             public BRCryptoFeeBasis feeBasis;
+            public boolean success;
```

Comment:
This should be `int`. This described the fields of the struct so must match the underlying C struct. Based on our recent discussion about boolean usage, this should be changed.

Reply:
Done.